src/Markdown.hs

```diff
@@ -26,14 +16,23 @@ instance ToMarkup Markdown where
     -- | Sanitized by default
     toMarkup = handleError . markdownToHtml
 
-strippedText :: ToMarkup a => a -> Text
-strippedText = pack . runLA
-    (   arr toHtml
-    >>> arr renderMarkup
-    >>> hread
-    //> getText
-    >>. concat
-    )
+toPandoc :: Markdown -> Pandoc
+toPandoc m = either mempty id $ parse m
```

Comment:
Hmm, I can futz in ghci a bit later, but I see `f m = ... $ g m` and assumed `f = ... . g` would work. No dice?